### PR TITLE
Bump version to 0.2.0

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,16 +37,14 @@ about:
...
-    This is a small library to manage continuous streams of data, particularly when complex branching and control flow situations arise.
+    Manage streaming data, optionally with Dask and Pandas
   doc_url: https://streamz.readthedocs.io/en/latest/
   dev_url: https://github.com/mrocklin/streamz/
 
 extra:
   recipe-maintainers:
     - CJ-Wright
     - mrocklin
-    - ordirules
```
Comment (CJ-Wright) [inline recipe/meta.yaml:51]: I think he goes by @jrmlhermitte now

File: recipe/meta.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: :+1:

File: recipe/meta.yaml (hunk 1 above)
Comment (jrmlhermitte) [reply recipe/meta.yaml:51]: oh yes sorry about change and thanks @CJ-Wright

**Comment by mrocklin** [comment]
The build exception here confuses me.  I don't know where the other SHA came from.  This PR is consistent with the sha on pypi.org.

**Comment by mrocklin** [comment]
Or so I believe

**Comment by CJ-Wright** [comment]
The problem is that were getting the package from GitHub not pypi

**Comment by mrocklin** [comment]
That would do it :)

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/{{ org }}/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: I think you may need this script still to do the install.

**Comment by mrocklin** [comment]
Thanks for the help @CJ-Wright

**Comment by CJ-Wright** [comment]
Of course @mrocklin

**Pull request merged**